packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -128,6 +129,23 @@ const ProductFullDetail = props => {
         }
     }
 
+    let customAttributesDetails = [];
+    const customAttributesDetailsPageBuilder = [];
+    if (Array.isArray(customAttributes)) {
+        customAttributes.forEach(customAttribute => {
+            if (
+                customAttribute.attribute_metadata.ui_input.ui_input_type ===
```

Comment:
Are we sure customAttribute.attribute_metadata.ui_input.ui_input_type is always available? This code here https://github.com/magento/magento2-pwa/blob/main/CatalogGraphQlAux/Model/Resolver/CustomAttributes.php#L104 would indicate to me that it could well be not set.

Reply:
Hi @fooman this this field value is resolved here https://github.com/magento/magento2-pwa/blob/main/EavGraphQlAux/Model/Resolver/DataProvider/AttributeMetadata.php#L89